IframeEmbedding/IframeEmbedding.csproj.user

```diff
@@ -0,0 +1,38 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is unwanted and dynamically creating file. Remove this.

Reply:
Removed this file